app/matplotlib.ipynb

```diff
@@ -86,23 +101,37 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
    "source": [
     "x = np.random.rand(50)\n",
     "y = np.random.rand(50)\n",
     "colors = np.random.rand(50)\n",
     "sizes = 100 * np.random.rand(50)\n",
     "\n",
-    "# write your code here"
+    "plt.scatter(x, y, c=colors, s=sizes)\n",
+    "plt.xlabel(\"Random X\")\n",
+    "plt.ylabel(\"Random Y\")\n",
+    "plt.show()"
```

Comment:
In the scatter plot task, you have not set the colorbar label, which is part of the task requirements. Consider adding a colorbar and setting its label to provide more information about the plot.